### PR TITLE
Add stat graphic and some model changes

### PR DESCRIPTION
**Pull request by afieldi**
Pulls a weeks worth of games and creates a weekly report:
image

### CODE REVIEW

File: BackEnd/src/business/teams.ts [hunk 1]
```diff
@@ -44,10 +44,12 @@ export async function GetTeamsBySeasonId(seasonId: number): Promise<TeamModel[]>
 
 export async function buildRisenTeams() {
   let sheetName = 'Teams and Standings';
-
+  console.log("I AM HERE");
```
Comment (Vexrax) [inline BackEnd/src/business/teams.ts:47]: Debug log

File: BackEnd/src/business/teams.ts [hunk 2]
```diff
@@ -44,10 +44,12 @@ export async function GetTeamsBySeasonId(seasonId: number): Promise<TeamModel[]>
...
   let seasonsWithSheets = await GetDbActiveSeasonWithSheets();
+  console.log(seasonsWithSheets);
```
Comment (Vexrax) [inline BackEnd/src/business/teams.ts:49]: debug log

File: BackEnd/src/business/teams.ts [hunk 3]
```diff
@@ -44,10 +44,12 @@ export async function GetTeamsBySeasonId(seasonId: number): Promise<TeamModel[]>
...
   for (let seasonsWithSheet of seasonsWithSheets) {
     let sheet = await GetGoogleSheet(seasonsWithSheet.googleSheetId, sheetName);
+    logger.info(sheet);
```
Comment (Vexrax) [inline BackEnd/src/business/teams.ts:52]: Dont think we need this

File: front-end/src/components/banner/player-champ-banner.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    return (
+      <Box sx={{ height: 500 }}>
+        <Box>
+          <Typography align='left' fontFamily="Montserrat">{position}</Typography>
+        </Box>
+        <Box sx={{ height: 300, overflow: 'hidden', width: 190, clipPath: 'polygon(0 0, 75% 0, 100% 100%, 25% 100%)', backgroundImage: `url(/images/champions/profile/${championPortraitId}_0.jpg)`, backgroundSize: 'cover', backgroundPosition: 'center' }}>
+          {/* <ImgBox
+            src={'/images/champions/splash/1_0.jpg'}
+            height={'400px'}
+          /> */}
```
Comment (Vexrax) [inline front-end/src/components/banner/player-champ-banner.tsx:40]: Commented out code, should remove if we dont need it

File: front-end/src/common/stats-generators/game-rating/RoleRatingStatGenerator.ts [hunk 5]
```diff
@@ -35,4 +35,10 @@ export abstract class RoleRatingStatGenerator extends GameRatingStatGenerator {
   abstract getJunglerStatValue(playerStatsModel: AggregatedPlayerStatModel): number;
 
   abstract getSupportStatValue(playerStatsModel: AggregatedPlayerStatModel): number;
+
+  // abstract getSoloLaneStatValueGameModel(playerGameModels: PlayerGameModel[]): number;
+
+  // abstract getJunglerStatValueGameModel(playerGameModels: PlayerGameModel[]): number;
+
+  // abstract getSupportStatValueGameModel(playerGameModels: PlayerGameModel[]): number;
```
Comment (Vexrax) [inline front-end/src/common/stats-generators/game-rating/RoleRatingStatGenerator.ts:43]: Commented out code, if we dont need it lets remove

Comment (afieldi) [reply front-end/src/common/stats-generators/game-rating/RoleRatingStatGenerator.ts]: Fixed

File: front-end/src/components/banner/player-champ-banner.tsx (hunk 4 above)
Comment (afieldi) [reply front-end/src/components/banner/player-champ-banner.tsx]: Fixed

File: BackEnd/src/business/teams.ts (hunk 3 above)
Comment (afieldi) [reply BackEnd/src/business/teams.ts:52]: Fixedd

Comment (Vexrax) [reply BackEnd/src/business/teams.ts:52]: Am i missing something or is this logger line still here are the "fixed" commit

**Review by Vexrax — APPROVED** [review]
LGTM

**Pull request merged**